machine/v1beta1/types_awsprovider.go

```diff
@@ -165,6 +165,11 @@ type EBSBlockDeviceSpec struct {
 	// it is not used in requests to create gp2, st1, sc1, or standard volumes.
 	// +optional
 	Iops *int64 `json:"iops,omitempty"`
+	// Throughput to provision in MiB/s. This parameter is valid only for gp3 volumes.
+	//
+	// Constraints: 125-1000 for General Purpose SSD (gp3)
```

Comment:
can we validate this constraints with min/max and that this is only set when VolumeType=gp3 with CEL

Reply:
can we also doc what's default if unset and since this is mutable what happens if changed day2?